openlibrary/templates/type/edition/view.html

```diff
@@ -364,8 +364,7 @@ <h3 class="list-header collapse">$_("ID Numbers")</h3>
     <script type="text/javascript">
     <!--
     window.q.push(function(){
-        \$('#historyTools').append('| <a href="javascript:;" id="wikilink" title="Cite this on Wikipedia">Wikipedia citation</a>');
-        \$('#wikilink').colorbox({inline:true,opacity:"0.5",href:"#wikicode"});
```

Comment:
This line wasn't moved to the JS

Reply:
Maybe it's an issue with the HTML being added dynamically?

Reply:
window.q runs AFTER the JS has loaded so this makes sense.

Easiest fix is to move this to the index.js and append to #historyTools whenever a historyTools element is present. Sound good?

Reply:
This shouldn't be using JS at all; I moved the HTML into lib/history here https://github.com/jdlrobson/openlibrary/pull/1

Reply:
Resolved by @cdrini